src/gui/src/displayControls.cpp

```diff
@@ -381,6 +391,36 @@ DisplayControls::~DisplayControls()
   custom_controls_.clear();
 }
 
+void DisplayControls::createLayerMenu()
+{
+  connect(layers_menu_->addAction("Only show selected"),
+          &QAction::triggered,
+          [this]() {
+            layerShowOnlySelectedNeighbors(0, 0);
+          });
+  auto* neighbors = layers_menu_->addMenu("Show with neighboring layers");
```

Comment:
Having these in a submenu makes it a bit slower to use.  I think it would be better to just put them in the main popup menu.